tests/certification/state/memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# Memcached State Store certification testing
+
+This project aims to test the [Memcached State Store] component under various conditions.
+
+This state store [supports the following features][features]:
+* CRUD
+* TTL
+
+# Test plan
+
+## Basic Test for CRUD operations:
+1. Able to create and test connection.
+2. Able to do set, fetch, update and delete.
+3. Negative test to fetch record with key, that is not present.
+
+## Test save or update data with different TTL settings:
+1. TTL not expiring ([`0` for memcached](https://github.com/memcached/memcached/wiki/Commands#set))
+2. TTL not a valid number
+3. Provide a TTL of 1 second:
+    1. Fetch this record just after saving
+    2. Sleep for 2 seconds
+    3. Try to fetch again after a gap of 2 seconds, record shouldn't be found
+
```

Comment:
Please add a test where you:
1. Set a key to show the connection is fine. Make the TTL 2 minutes or something.
2. Interrupt the network (the memcache ports) for say 60 seconds or something.
3. Wait 10 seconds.
4. Try to read the key you wrote earlier and assert the value.

Reply:
fixed in d3a7b524976cad00a6023fb342263dfeafc94093